.editorconfig

```diff
@@ -7,7 +7,7 @@ trim_trailing_whitespace = true
 insert_final_newline = true
 end_of_line = lf
 indent_size = 2
-max_line_length = 120
+max_line_length = 400
```

Comment:
I know that prettier and eclint do not use this configuration in the same way, for that reason we have a [specific configuration for prettier](https://github.com/FromDoppler/doppler-sms-mfe/blob/main/.prettierrc#L2).

I understand that it could still have issues, so, maybe this fix is fine, but [let us research a little more](https://makingsense.slack.com/archives/D01F11FDNAK/p1669637996838059) to understand what happened.

Reply:
@mcappato confirmed that it is because of the `package-lock.json` file. 

You could remove that file (if you decide to use _Yarn_), or add it to the exceptions section in `.editorconfig` file if you decide to continue using _NPM_ in place of _Yarn_.

Reply:
It is fixed, resolving the conversation...